deep_agent/aegra/mcp_auth.py

```
@@ -136,6 +145,70 @@ async def _resolve_sso(self) -> str:
+                mcp_name,
+                exc_info=True,
+            )
+            raise RuntimeError(
+                f"MCP '{mcp_name}': client_credentials token acquisition failed"
+            )
+
+        access_token = body.get("access_token")
+        if not isinstance(access_token, str) or not access_token:
+            raise RuntimeError(
+                f"MCP '{mcp_name}': client_credentials response missing access_token"
+            )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)